docs/rbac.rst

```diff
@@ -0,0 +1,269 @@
+Role Based Access Control
+=========================
+
+.. important::
+
+    This feature is a Tech Preview and can be subject of change.
```

Comment:
This should be rephrased as "and *may* be subject *to* change"

I'm not really sure how to express the rule about "may" vs "can" but "may" is a little more formal and fits better in this situation.